packages/introspect/src/e2e.test.ts

```diff
@@ -81,4 +99,74 @@ describe('e2e', () => {
                 }),
         });
     }, 100_000);
+
+    test('Add to Pagila', async () => {
+        await deleteData();
+
+        await createTable();
+
+        await insertData();
+
+        for (const [name, value] of Object.entries(await selectData())) {
+            console.log({
+                'Name:': name,
+                'Value:': value,
+                'JS primitive type': typeof value,
+            });
+        }
+
+        await deleteData();
+    }, 100_000);
 });
+
+const createTable = async () => {
+    try {
+        await client.connect();
+
+        await client.query(createTableQuery);
+
+        console.log('Table created successfully!');
+    } catch (err) {
+        console.error('Error creating table', err);
+    }
+};
+
+const insertData = async () => {
+    try {
+        await client.connect();
+
+        await client.query(insertTableQuery);
+
+        console.log(`Record inserted successfully!`);
+    } catch (err) {
+        console.error('Error inserting record', err);
+    }
+};
+
+const selectData = async () => {
+    try {
+        await client.connect();
+
+        const results = await client.query(selectTableQuery);
+
+        console.log(`Record found successfully!`);
+
+        return results.rows[0];
+    } catch (err) {
+        console.error('Error finding record', err);
+    }
+};
+
+const deleteData = async () => {
```

Comment:
this file is breaking the code co-location principle @jimezesinachi :

You're basically splitting functionality that is dependend on each other into two separate files. You should keep functionality that depends on each other co-located.

Reply:
Honestly I would just put everything in the same file and improve the name of the file. 

We should call it something along the lines of "test serialisation of all pg types"